base_layer/wallet_ffi/src/callback_handler.rs

```diff
@@ -238,7 +235,7 @@ where TBackend: TransactionBackend + 'static
             callback_contacts_liveness_data_updated,
             callback_balance_updated,
             callback_transaction_validation_complete,
-            callback_saf_messages_received,
+            _callback_saf_messages_received: callback_saf_messages_received,
```

Comment:
We may want to use this in future - removing it will require the mobile wallets to update their call
If we implement SAF in future they will need re add it - I think it's fine to keep the public interface the same and just never call the call back in the interim.